Theory assessment/Theory_assessment_Software_Engineering.md

```diff
@@ -0,0 +1,150 @@
+### 1. How does Object-oriented Programming differ from Process Oriented Programming? 
+A: Object-oriented programming is divided into the objects, but process-oriented programming is divided into smaller functions. OOP has bottom-up approach and data members can be public/private/default, which can also help hide the data and is more secure. POP has top-to-down approach and the data members are public, also data hiding is impossible. 
```

Comment:
8/10 The main characteristic of OOP is not clearly described. i.e. classes for data encapsulation and the objects as instances of the class.